WRidder/react-spa
# [markdown] make markdown parsing closer to github's

Summary:
By default, simple-markdown ends rules with a blank line
(two newlines). Github allows ending most rules with a
single newline, which affects headings in the README here.

This commit removes the double newline requirement from
the heading parsing. It also adds an extra newline around
either side of the code block so that it parses as a block
rather than an inline code element, and makes the markdown
output function output <p> tags instead of
<div class="paragraph"> tags.

Test Plan:
1. run `npm install`
2. run `cd server && node src/server.js`
3. load http://localhost:8080/ and http://localhost:8080/about
4. compare their output; see that the headings and code
   look the same in both, and that the output matches at
   various points in each page.
